docs/components/TextFields.md

```diff
@@ -132,13 +136,45 @@ filledTextField.editText?.doOnTextChanged { inputText, _, _, _ ->
 }
 ```
 
+### Filled password text field
+
+Filled password text fields is a filled text field implementation that includes password visual transformation and optional visualisation icon.
```

Comment:
There is typo, this should be `Filled password text field is`.

Same comment for `Outlined password text field` paragraph.